Automattic/wp-calypso
# My Home: Update copy in the growth summit card

#### Changes proposed in this Pull Request

* Update the copy in the growth summit card
![image](https://user-images.githubusercontent.com/13596067/127091585-03834318-8b12-4d93-a2d4-8488063440ad.png)

#### Testing instructions



* Apply D64596-code for your sandbox
* Go to My Home
* Check you can see the card and the copy is correct



Close https://github.com/Automattic/wp-calypso/issues/54850

#### TODO

- [x] Confirm the pager would be the better choice or not
- [x] Confirm should show to non-English user or not
- [x] Confirm when is the last time user can see the card
